.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+npm run lint-staged
```

Comment:
This is another change related to Prettier. You can ignore it. You can just mark this comment as resolved.